File: loops/FizzBuzz.c
#include <stdio.h>
char fizz[] = "fizz";
char buzz[] = "buzz";
char fizzbuzz[] = "fizzbuzz";
// first block of code
int main() {

    int i = 0;
// for loop
    for(i=1;i<=50;i++){
// if statments
        if(i%3==0 && i%5==0){
            printf("%s\n", fizzbuzz);
        }
        else if(i%3==0){
            printf("%s\n", fizz);
        }

        else if(i%5==0){
            printf("%s\n", buzz);
        }
        else{
            printf("%d\n", i);
        }
    }
}

File: loops/FizzBuzz.py
#loopthat counts to 50
i = 0 
while i<=49:
#increases i by one each loop
    i+=1
    if (i%3==0 and i%5==0):
        print("fizzbuzz")
    elif (i%3==0):
        print("fizz")
    elif (i%5==0):
        print("buzz")
    else:
        print(i)

